CHANGELOG.md

```diff
@@ -15,12 +15,14 @@ Note: Changes are now automatically tracked in [GitHub](https://github.com/sqlfl
 
 ### Changed
 - Fix typo in the in the wild page [#1285](https://github.com/sqlfluff/sqlfluff/pull/1285)
+- Moved the dbt templater into a plugin rather than part of the core. This should now be installed
```

Comment:
With the addition of the GitHub action, CHANGELOG no longer needs updating prior to a release.

Reply:
Removed during merging to `main`. Resolving.